### ISSUE TITLE
pipe CLI shall not fail with --skip-existing option

### ISSUE THREAD

**Issue by ekazachkova** [epam/cloud-pipeline#1903]
**Describe the bug**
The current version of pipe CLI fails during `pipe storage cp/mv` operation with enabled `-s (--skip-existing)` 

**To Reproduce**
1. Upload file 'test.txt' to bucket
2. Try to upload file via pipe CLI with enabled `-s` option: `pipe storage cp test.txt s3://test-bucket/test.txt -s -f`
3. See error:
```
$ pipe storage cp test.txt s3://test-bucket/test.txt -s -f
Skipping file /path/to/file.txt since it exists in the destination test.txt
Error: 'NoneType' object has no attribute 'source_key'
```

**Expected behavior**
```
$ pipe storage cp test.txt s3://test-bucket/test.txt -s -f
Skipping file /path/to/file.txt since it exists in the destination test.txt
```

**Environment:**
 - Cloud Provider: AWS
 - Version 0.17.0.9153

**Additional context**
The same error for folders copy/move (with `--recursive (-r)` option)

**Comment by tcibinan** [comment]
Should be fixed by af35bce097c33c6d386b953a998b49629e379e16. @ekazachkova Could you please verify?

**Comment by ekazachkova** [comment]
@tcibinan Verified, works well. Thanks!
